layouts/specifications/list.html

```diff
@@ -12,23 +12,15 @@
     <h1 class="big-text">
       {{ .Title }}
       {{ if .Params.project_id }}
-        <span class="badge">
-          <a href="https://projects.eclipse.org/projects/{{ .Params.project_id }}"><i class="fa fa-archive" aria-hidden="true"></i> Project</a>
+        <span class="eclipsefdn-specification-badges" data-project-id="{{ .Params.project_id }}">
```

Comment:
Not sure why we are duplicating this 3 time in this file but I believe you need to add this widget in two more places before we can merge! Thanks for working on this!

Reply:
Np!

That's a good catch. I didn't notice it being used in multiple places. I added the possibility of having a video guide badge to the widget.